lib/view_component/polymorphic_slots.rb

```diff
@@ -42,6 +42,10 @@ def register_polymorphic_slot(slot_name, types, collection:)
           define_method(getter_name) do
             get_slot(slot_name)
           end
+
+          define_method("#{getter_name}?") do
+            public_send(getter_name).present?
```

Comment:
I think we want to avoid `public_send` here, since it's likely slower than calling methods directly.

Could we use `get_slot` instead of `public_send`?

Reply:
Ah, I suppose that's important when it comes to ViewComponent. TIL!

Reply:
Reverted back to using `get_slot`